x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/components/search_ui_graphic.scss

```diff
@@ -0,0 +1,188 @@
+.search-ui-graphic {
```

Comment:
[super optional, please feel free to skip if it's too much of a pain] I honestly don't really mind it in this case since it's basically porting over the existing SVG/CSS as-is, but if you felt up for it we could convert all the CSS classes in here to `camelCasing` to match Kibana/EUI

Reply:
Ohhhh shoot, I will do that.